ios/sdk/WeexSDK/Sources/Module/WXWebSocketModule.m

```diff
@@ -126,7 +126,7 @@ - (void)close
 
 - (void)close:(NSString *)code reason:(NSString *)reason
 {
-    if(!code)
+    if([WXUtility isBlankString:code])
```

Comment:
if code = nil ,has any question?

Reply:
there is a assert crash  when code is empty string but not a nil value